src/outputs/outputs_main.py

```diff
@@ -34,6 +40,9 @@ def run_output(
     # Creating blank columns for short form output
     short_form_df = run_shortform_prep(estimated_df, round_val=4)
 
+    # Create combined ownership column using mapper
+    short_form_df = combine_dataframes(short_form_df, ultfoc_mapper)
```

Comment:
Hi Vondy, we need to combine your mapper to the estimated_df before we do any processing for the short form.
This is because we will need the extra column for the long form and the short form.

I'm not sure where your "combine_dataframes" function should go. I wonder if it's worth having another script called outputs/outputs_prep.py where we do a few steps like yours that will be used for all forms?